### PR TITLE
Add SemVer 2.0.0 italian translation

### PR DESCRIPTION
**Pull request by taringamberini**
This is the italian translation of Semantic Versioning 2.0.0.

With this translation comes little improvements in language support.

Accordingly to [The Internationalization Best Practices: Specifying Language in XHTML & HTML Content](http://www.w3.org/TR/2007/NOTE-i18n-html-tech-lang-20070412/) I've improved support language in `_layouts` files:
- adding `lang` and `dir` attributes to `html` tag
- adding `lang`, `rel` and `hreflang` attributes to the `a` tags that contains a language name
- making a trade off between [Best Practice 8: Handling attribute values and element content in different languages](http://www.w3.org/TR/2007/NOTE-i18n-html-tech-lang-20070412/#ri20050128.175100333)  and [not annoying translators with name language](https://github.com/omasanori/semver.org/commit/7626a17c6986a47eb383d591869f4680c133ea53#commitcomment-3736397) achieving a solution like the one on the top of the page at [gnu.org](http://www.gnu.org/) with language code after the localized language name.

Thanks to @nafigator and @omasanori for feedback.

Let me know if you like these new little improvements.

Thank you!

### CODE REVIEW

**Comment by taringamberini** [comment]
The review has been completed: the Italian translation is ready.

I hope you like the little improvements in language support I have introduced. I hope I did things right too, because I could not test how the layout will be rendered in a browser.

Is there something else I should do?

Thank you,
Tarin

**Comment by taringamberini** [comment]
The review has been completed.

Is there something else I should do in order to get the italian translation merged?

Thank you,
Tarin Gamberini

File: README [hunk 1]
```diff
@@ -1 +1,3 @@
-This is the site at http://semver.org/ that describes the Semantic Versioning specification.
\ No newline at end of file
+This is the site at http://semver.org/lang/it that translates in Italian the Semantic Versioning specification.
```
Comment (haacked) [inline README]: Why this change? This is the README for semver.org and not _just_ the italian version. :smile:

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 _site
 *.swp
+/nbproject/
```
Comment (haacked) [inline .gitignore]: What is this?

File: CNAME [hunk 3]
```diff
@@ -1 +1 @@
-semver.org
+it.semver.org
```
Comment (haacked) [inline CNAME]: Please revert this change.

**Comment by haacked** [comment]
Looks good apart from the two changes I requested. :)

**Comment by taringamberini** [comment]
> Looks good apart from the two changes I requested. :)

Sorry for flaws I have made, I hope it is all right now.

Many thanks :)

**Comment by haacked** [comment]
grazie!

thumbs-up-keanu
